### PR TITLE
[FEAT] custom tokens push notifications

### PR DESCRIPTION
**Pull request by gabrielbazan7**

### CODE REVIEW

**Review by rastajpa — APPROVED** [review]
Nice work! 
Screenshot_2021-09-09-13-10-40-094_com miui home

**Review by cmgustavo — DISMISSED** [review]
ACK!

File: packages/bitcore-wallet-service/src/lib/pushnotificationsservice.ts [hunk 1]
```diff
@@ -397,17 +397,33 @@ export class PushNotificationsService {
...
             unit = Constants.TOKEN_OPTS[tokenAddress].symbol.toLowerCase();
             label = UNIT_LABELS[unit];
           } else {
-            label = 'tokens';
-            throw new Error('Notifications for unsupported token are not allowed');
+            let customTokensData;
+            try {
+              customTokensData = await this.getTokenData();
+            } catch (error) {
+              throw new Error('Notifications for unsupported token are not allowed');
```
Comment (kajoseph) [inline packages/bitcore-wallet-service/src/lib/pushnotificationsservice.ts:411]: Could you change this to be different than the error message below? It makes it harder to debug if the same error is thrown in multiple places.

**Review by kajoseph — CHANGES_REQUESTED** [review]

**Review by kajoseph — APPROVED** [review]
LGTM

**Pull request merged**